views/index.handlebars

```diff
@@ -100,6 +100,13 @@
       </div>
     </div>
 
+    <div id="compliance-error">
+      <div class="title">
+        Encryption error<br>
+        Your browser does not support gcm encryption.
```

Comment:
Which browser/versions would this apply to?

It feels like this should be slightly more actionable (like prompting the user to the Download Firefox page or something), but maybe that is contained in the new UI specs.